### PR TITLE
refactor: Convert Tag stories to csf format

### PR DESCRIPTION
**Pull request by brianacnguyen**
## **Description**
- Converted Tag and TagUrl stories to new format
- Added sample const to TagUrl
- Updated tag height to 24px by design. This also fixes any tag cutoff issues

## **Screenshots/Recordings**
Simulator Screen Shot - iPhone 14 Pro - 2023-10-10 at 19 00 02
Simulator Screen Shot - iPhone 14 Pro - 2023-10-10 at 19 01 24
Simulator Screen Shot - iPhone 14 Pro - 2023-10-10 at 19 01 26
Simulator Screen Shot - iPhone 14 Pro - 2023-10-10 at 19 01 29
Simulator Screen Shot - iPhone 14 Pro - 2023-10-10 at 19 01 31

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained:
  - [ ] What problem this PR is solving.
  - [ ] How this problem was solved.
  - [ ] How reviewers can test my changes.
- [ ] I’ve indicated what issue this PR is linked to: Fixes #???
- [ ] I’ve included tests if applicable.
- [ ] I’ve documented any added code.
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by tommasini — DISMISSED** [review]
LGTM, just a curious question, I can see on the screenshots the tagUrl is not aligned on the center, is that intended?

File: .storybook/main.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ module.exports = {
   stories: [
     '../app/component-library/components/Cards/Card/**/*.stories.?(ts|tsx|js|jsx)',
     '../app/component-library/components/Banners/Banner/variants/BannerAlert/**/*.stories.?(ts|tsx|js|jsx)',
+    '../app/component-library/components/Tags/**/*.stories.?(ts|tsx|js|jsx)',
```
Comment (georgewrmarshall) [inline .storybook/main.js:5]: Can we update this folder path so it's less specific and doesn't create merge conflicts every time a story is created?

```
stories: [
    '../app/components/**/*.stories.?(ts|tsx)',
    '../app/component-library/**/*.stories.?(ts|tsx)',
  ],
```

File: app/component-library/components/Tags/TagUrl/TagUrl.stories.tsx [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
+  argTypes: {
+    label: {
+      control: { type: 'text' },
+      defaultValue: SAMPLE_TAGURL_PROPS.label,
+    },
+    ctaLabel: {
+      control: { type: 'text' },
+      defaultValue: SAMPLE_TAGURL_PROPS.cta?.label,
+    },
+  },
```
Comment (georgewrmarshall) [inline app/component-library/components/Tags/TagUrl/TagUrl.stories.tsx:24]: Missing `imageSource` as an argtype?

File: app/component-library/components/Tags/TagUrl/TagUrl.stories.tsx [hunk 3]
```diff
@@ -1,30 +1,38 @@
...
+
+const TagUrlMeta = {
+  title: 'Component Library / Tags',
+  component: TagUrlComponent,
+  argTypes: {
+    label: {
+      control: { type: 'text' },
+      defaultValue: SAMPLE_TAGURL_PROPS.label,
+    },
+    ctaLabel: {
```
Comment (georgewrmarshall) [inline app/component-library/components/Tags/TagUrl/TagUrl.stories.tsx:20]: I think the controls should consistently reflect the props of the component but I see that the `cta` is an object. Could we use dot notation for args that are keys for an object prop?
```suggestion
   'cta.label': {
```
Screenshot 2023-10-24 at 1 07 15 PM

**Review by georgewrmarshall — DISMISSED** [review]
Looking good! Left a few non-blocking comments.
- pulled branch and checked stories and controls

Also, looks like the alignment is off on the TagUrl might be a quick fix to add it here?

Screenshot 2023-10-24 at 1 03 36 PM

File: .storybook/main.js (hunk 1 above)
Comment (georgewrmarshall) [reply .storybook/main.js:5]: As @brianacnguyen pointed out. Storybook always will create conflicts with the `.storybook/storybook.requires.js` file anyway

**Review by georgewrmarshall — COMMENTED** [review]
Need to resolve merge conflicts

**Review by georgewrmarshall — APPROVED** [review]
LGTM! I'm being overly cautious with some of those comments

File: app/component-library/components/Cells/Cell/__snapshots__/Cell.test.tsx.snap [hunk 4]
```diff
@@ -215,7 +215,7 @@ exports[`Cell should render CellDisplay given the type Display 1`] = `
             "borderColor": "#BBC0C5",
             "borderRadius": 10,
             "borderWidth": 1,
-            "height": 20,
+            "height": 24,
```
[the remark this answers is not in the document]
Comment (brianacnguyen) [reply app/component-library/components/Cells/Cell/__snapshots__/Cell.test.tsx.snap:218]: yes. to sync with design

**Pull request merged**